src/oas3/operation.ts

```diff
@@ -38,7 +39,8 @@ export const transformOas3Operation: Oas3HttpOperationTransformer = ({ document,
     method,
     path,
     summary: operation.summary,
-    responses: translateToResponses(operation.responses),
+    // fixme: this doesn't make sense, since the array could be empty
```

Comment:
@XVincentX we chatted about this on Slack.
I'm leaning towards making a change in `@stoplight/types` to accept any array of `IHttpOperationResponse`, but curious to hear your thoughts.

Reply:
+1 for updating @stoplight/types to be more lenient